setup.py

```diff
@@ -73,6 +73,7 @@
     "gui-traitsui": ["hyperspy_gui_traitsui>=1.1.0"],
     "mrcz": ["blosc>=1.5", 'mrcz>=0.3.6'],
     "speed": ["numba"],
+    "usid": ["pyUSID"],
```

Comment:
I should be fine to add `pyUSID` as required dependency since it is available from the conda-forge channel and on pypi. The only major drawback is that it will increase our numpy version requirement to >1.13.

Reply:
We use only one feature of Numpy 1.13 (can't remember which one) which is why we had to bump up our minimum version number. I can see how we can work around this issue. 

I currently listed pyUSID only as an optional requirement. Did you mean to include it as a core requirement? Is there a reason why bumping numpy requirement to 1.13 would be a major drawback / show-stopper?

Reply:
The latest version of pyUSID (0.0.6.1) now only needs numpy 1.10. This should alleviate your concern